tests/pytest/test.py

```diff
@@ -544,6 +544,26 @@ def testArrIndexCommand(env):
     r.assertEqual(r.execute_command('JSON.ARRINDEX', 'test', '.arr', 2, 3), 5)
     r.assertEqual(r.execute_command('JSON.ARRINDEX', 'test', '.arr', '[4]'), 4)
 
+def testArrInsertCommand(env):
+    """Test JSON.ARRINSERT command"""
+    r = env
+
+    r.assertOk(r.execute_command('JSON.SET', 'test', '.', '{ "arr": [] }'))
+    r.assertEqual(r.execute_command('JSON.ARRINSERT', 'test', '.arr', 0, '1'), 1)
+    r.assertEqual(r.execute_command('JSON.ARRINSERT', 'test', '.arr', -1, '2'), 2)
+    r.assertEqual(r.execute_command('JSON.ARRINSERT', 'test', '.arr', -2, '3'), 3)
+    r.assertEqual(r.execute_command('JSON.ARRINSERT', 'test', '.arr', 3, '4'), 4)
+
+    r.assertEqual(r.execute_command('JSON.GET', 'test', '.arr'), "[3,2,1,4]")
+
+    r.assertEqual(r.execute_command('JSON.ARRINSERT', 'test', '.arr', 1, '5'), 5)
+    r.assertEqual(r.execute_command('JSON.ARRINSERT', 'test', '.arr', -2, '6'), 6)
+    
+    r.assertEqual(r.execute_command('JSON.GET', 'test', '.arr'), "[3,5,2,6,1,4]")
+
+    r.expect('JSON.ARRINSERT', 'test', '.arr', -7, '6').raiseError()
+    r.expect('JSON.ARRINSERT', 'test', '.arr', 7, '6').raiseError()
```

Comment:
Suggest to add coverage also for multi json values, e.g., add
```
r.assertEqual(r.execute_command('JSON.ARRINSERT', 'test', '.arr', -3, '7', '8', '9'), 9)
r.assertEqual(r.execute_command('JSON.GET', 'test', '.arr'), "[3,5,2,7,8,9,6,1,4]")
```
Or even with a more complex value
```
r.assertEqual(r.execute_command('JSON.ARRINSERT', 'test', '.arr', -3, '7', '{"A":"Z"}', '9'), 9)
r.assertEqual(r.execute_command('JSON.GET', 'test', '.arr'), "[3,5,2,7,{"A":"Z"},9,6,1,4]")
```